### ISSUE TITLE
Proposal for Plugin Documentation

### ISSUE THREAD

**Issue by macanudo527** [eprbell/dali-rp2#29]
I noticed CCXT documents a sample REST response for each of their calls. I propose we do the same for our plugins where needed. For example, I added the following to the plugin I'm working on:

```
fiat_payments = client.sapiGetFiatPayments(params=({'transactionType':0, 
	'beginTime':(int(startTime.timestamp()) * 1000), 
	'endTime':nowTime}))
# {
#    "code": "000000",
#    "message": "success",
#    "data": [
#    {
#       "orderNo": "353fca443f06466db0c4dc89f94f027a",
#       "sourceAmount": "20.0",  // Fiat trade amount
#       "fiatCurrency": "EUR",   // Fiat token
#       "obtainAmount": "4.462", // Crypto trade amount
#       "cryptoCurrency": "LUNA",  // Crypto token
#       "totalFee": "0.2",     // Trade fee
#       "price": "4.437472", 
#       "status": "Failed",  // Processing, Completed, Failed, Refunded
#       "createTime": 1624529919000,
#       "updateTime": 1624529919000  
#    }
#    ],
#    "total": 1,
#    "success": true
# }
``` 

Fiat Payments responds with that json. I think this will make it easier to sort out what variables we need and also document undocumented APIs like what happened with Coinbase plugin.  However, it might bloat the code. Any thoughts?

**Comment by eprbell** [comment]
More documentation: sounds good to me. If we discover that in the end the Python file is too bloated we can always move the JSON snippet to a plugin-specific section of the docs. But capturing the information is good. Only, please redact any personal identifiable information from log snippets either in the code or on here (order numbers, dates, etc.).

**Comment by eprbell** [comment]
One more observation here: it would be good to also have unit tests for each of the documented snippets to limit the risk of regression.

**Comment by macanudo527** [comment]
I added unit tests for the [Binance.com plugin](https://github.com/macanudo527/dali-rp2/blob/6e17d754eb89f2594a8e194574e546a0caac7382/tests/test_plugin_binance_com.py) that incorporate the JSON snippets.

**Comment by macanudo527** [comment]
Should I add this to the [Plugin Laundry List](https://github.com/eprbell/dali-rp2/blob/main/README.dev.md#plugin-laundry-list), with an example?

**Comment by eprbell** [comment]
> 

Good idea, but try to keep the new bullet within a few lines length (say < 10), if possible.

**Comment by macanudo527** [comment]
Resolved with #70
